### PR TITLE
implement send_bits

### PR DESCRIPTION
**Pull request by CrowdHailer**
I couldn't really what to do regarding tests, so I haven't added any.
 Might be helpful to add doc links to the actual services used.

Please feel free to run with this. I'm using it locally and don't have strong opinions on naming

### CODE REVIEW

File: src/ffi.mjs [hunk 1]
```diff
@@ -36,6 +36,17 @@ export function to_fetch_request(request) {
...
+export function bitarray_request_to_fetch_request(request) {
+  let url = uri_to_string(to_uri(request));
+  let method = method_to_string(request.method).toUpperCase();
+  let options = {
+    headers: make_headers(request.headers),
+    method,
+  };
+  if (method !== "GET" && method !== "HEAD") options.body = request.body.buffer;
+  return new globalThis.Request(url, options);
+}
```
Comment (lpil) [inline src/ffi.mjs:48]: All the code here except `.buffer` is the same as the other function, let's extract that into one function.

**Review by lpil — COMMENTED** [review]
Thank you

File: src/ffi.mjs (hunk 1 above)
Comment (CrowdHailer) [reply src/ffi.mjs:48]: This looks better now

**Review by lpil — APPROVED** [review]
Thank you

**Pull request merged**